pkg/lifecycle/lifecycle.go

```diff
@@ -195,8 +195,8 @@ type ExpirationDate struct {
 
 // MarshalXML encodes expiration date if it is non-zero and encodes
 // empty string otherwise
-func (eDate *ExpirationDate) MarshalXML(e *xml.Encoder, startElement xml.StartElement) error {
-	if *eDate == (ExpirationDate{time.Time{}}) {
+func (eDate ExpirationDate) MarshalXML(e *xml.Encoder, startElement xml.StartElement) error {
+	if eDate.Time.IsZero() {
```

Comment:
This code is borrowed from server @vadmeste does this mean it needs to be fixed on server as well?

Reply:
> This code is borrowed from server @vadmeste does this mean it needs to be fixed on server as well?

Yes